Userland/Applications/ThemeEditor/ThemeEditor.gml

```diff
@@ -1,13 +1,42 @@
 @GUI::Widget {
-    layout: @GUI::HorizontalBoxLayout {}
+    layout: @GUI::VerticalBoxLayout {}
     fill_with_background_color: true
 
-    @GUI::Frame {
+    @GUI::Widget {
         layout: @GUI::HorizontalBoxLayout {}
-        name: "preview_frame"
+
+        @GUI::Frame {
+            layout: @GUI::HorizontalBoxLayout {}
+            name: "preview_frame"
+        }
+
+        @GUI::TabWidget {
+            name: "property_tabs"
+        }
     }
 
-    @GUI::TabWidget {
-        name: "property_tabs"
+    @GUI::Widget {
+        name: "theme_override_controls"
+        layout: @GUI::HorizontalBoxLayout {
+            margins: [ 0, 4 ]
+        }
+
+        fixed_height: 30
+
+        @GUI::Layout::Spacer {}
+
+        @GUI::Button {
+            name: "reset"
+            text: "Reset to previous System Theme"
```

Comment:
is "System Theme" capitalized like that elsewhere?

Reply:
It's not, weren't actions supposed to be named in book title case though? Maybe I misremember that

Reply:
Sure, but then you'll also have to capitalize 'previous' :)